tests/infographics/test_models.py

```diff
@@ -19,7 +19,7 @@ def test_create_success(self):
             infographic_type=InfographicType.LIST,
             data={
                 "is_numbered": True,
-                "items": [{
+                "objects": [{
```

Comment:
wouldn't these be called "entries"? It's also a bit less generic than "objects".